server/database/schemas/Forest.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const forestSchema = new Schema({
```

Comment:
Great work! the only thing missing it the "trees by type".
What we can do here is to save this in mongo
```
{
    ...
    forest_trees: [{
         name: "אלון",
         count: 5
     }]
    ...
}
```

Reply:
how we do that? read here, it's called subDocuments. https://mongoosejs.com/docs/subdocs.html
probablly something like this:

```
const treeSchema = new Schema({ name: {type: String}});

const forestSchema = new Schema({
  trees: [treeSchema]
});
```